org-roam.el

```diff
@@ -105,39 +125,81 @@ Valid states are 'visible, 'exists and 'none."
   (setq org-roam-files (deft-find-all-files))
   (async-start
    `(lambda ()
+      (load-file "~/.emacs.d/.local/straight/build/dash/dash.el")
       (require 'org)
+      (require 'dash)
```

Comment:
Let's avoid dash, I don't think it's used here anyway

Reply:
oops, should probably review my PRs a bit more careful.